### DISCUSSION TITLE
[Feature]关于强调色的显示优化

### DISCUSSION THREAD

**Discussion by Higanoneko** [Planshit/Tai#230] [Ideas]
![Snipaste_2023-07-08_18-42-52](https://github.com/Planshit/Tai/assets/50688679/dcd6c50e-4de4-4d8d-a744-711c698ca37f)
![Snipaste_2023-07-08_18-43-14](https://github.com/Planshit/Tai/assets/50688679/a27082df-6214-4e35-a07d-80625f7d88b4)
概览和详细页的强调色与设定颜色一致。
以及主题模式跟随系统

**Comment by noberumotto** [comment]
在1.5.0.3版本中优化了
